### ISSUE TITLE
Issue on docs?

### ISSUE THREAD

**Issue by Bardun** [elevenlabs/elevenlabs-docs#417]
Path: /voices/voice-lab/overview
In the documentation it says that I could "give voices in your VoiceLab (including those added from the Voice Library) custom names, descriptions, and tags."
This hasn't been the case for at least 2 weeks now.
As I only use the webservice, I don't know a thing about github - how and where to raise issues and stuff. I only know that I pay $100/month and now have to write down the names of voices I plan to use for various characters...

BTW: the new drop-down list of voices in the 'edit project' page is nice - but the function that updates the 'recently used' list doesn't work.

I appreciate your work, but it really s**ks if the usability gets worse with every update.
Please excuse any grammar or vocabulary errors - my first language is German (and I'm using your AI services to create German audio books).

thx

**Comment by louisjoecodes** [comment]
Hi @Bardun, thanks for flagging this.

Just to confirm I understand this request correctly, you are able to modify the labels for a voice:

![Screenshot 2024-10-21 at 15 21 53](https://github.com/user-attachments/assets/da8858fb-3549-4fb6-afbf-9516c4324aa1)


But you’re unable to filter by these labels in the voice search box?

![Screenshot 2024-10-21 at 15 20 36](https://github.com/user-attachments/assets/2300ef62-8b53-4c8c-ac22-03f937191bc4)

**Comment by Bardun** [comment]
No - I think I'm getting the wrong pop-up in my voices:
![image](https://github.com/user-attachments/assets/c6d865fc-63bf-450b-9c75-9afb7ef0c155)
This looks to me like the form to edit voices I have created myself, not the one for voices I clone to my list.
This started somewhere around the time when it became possible to add voices with one click. 

Searching voices works fine for me.

**Comment by louisjoecodes** [comment]
I see so just to confirm, has this issue been resolved? I notice that there are 6/5 labels so you may have to delete 2 to add one :)

**Comment by Bardun** [comment]
No, it has not been resolved - this is a voice that I have cloned to my list. 
I am not able to save any changes I make to that voice.
In fact, it is the first voice I encountered that problem with. I am able to make changes in the form - but when I hit the "edit voice" button nothing happens.

**Comment by louisjoecodes** [comment]
Understood thank you for clarifying.

**Comment by Bardun** [comment]
Thx for changing - I was able to 'rename' the voice I gave you as an example. Yet, I still can't do that with voice design voices :(
